pallets/protos/src/tests.rs

```diff
@@ -643,215 +643,215 @@ mod get_protos_tests {
 		});
 	}
 
-	#[test]
```

Comment:
whats going on here? why remove tests?
Should in fact add tests for the above change.